### PR TITLE
Update to Workout Creation

### LINKED ISSUE #15: Bug: Custom Workout Overwrite
**Describe the bug**
When creating a custom workout from WorkoutCreation.jsx, the previous workout is overwritten.

**To Reproduce**
1. Observe firebase db workouts doc
2. Create new workout
3. Note that previous data was overwritten

**Expected behavior**
The new workout data should be appended to the firebase workout doc

### PR DESCRIPTION
**Pull request by Turtleoo**
Closes: #15 
This update improves the functionality of the Workout Creation page by enabling users to create and save complete workout presets to Firebase. Instead of saving individual exercises separately, users can now group multiple exercises into a single workout preset (e.g., "Chest Day").

Full Workout Saving: 
- Users can now define a series of exercises and save them as a complete workout preset in Firebase.
Preset Recall:
- These presets can be easily recalled and performed anytime, allowing users to follow a structured routine for specific fitness goals.

### CODE REVIEW

**Comment by Turtleoo** [comment]
Changed updateDoc to setDoc, so a new user without a pre-existing set of workouts can create a workout without getting Firebase errors.

File: main/src/WorkoutCreation.jsx [hunk 1]
```diff
@@ -1,7 +1,16 @@
 import React, { useState } from 'react';
-import { Box, Typography, TextField, Button, Paper, List, ListItem, IconButton } from '@mui/material';
+import {
```
Comment (bishwasbhatt2) [inline main/src/WorkoutCreation.jsx]: just a style issue, if you can include this import state all in one line

**Review by bishwasbhatt2 — COMMENTED** [review]
just change the styling thats it

**Review by bishwasbhatt2 — APPROVED** [review]
lgtm

**Pull request merged**